src/pytest_cov/plugin.py

```diff
@@ -226,6 +244,13 @@ def pytest_runtestloop(self, session):
                 compat_session.testsfailed += 1
 
     def pytest_terminal_summary(self, terminalreporter):
+        if self._disabled:
+            markup = {'red': True, 'bold': True}
+            msg = (
+                'WARNING: Coverage disabled by user!'
```

Comment:
Ah yes, you're right. Still, I think it's good if we'd use the config.warn thing here ...